partitura/score.py

```diff
@@ -259,12 +258,10 @@ def measure_map(self):
             kind="previous",
             axis=0,
             fill_value="extrapolate",
+            dtype=int,
         )
 
-        def int_interp1d(input):
```

Comment:
these lines were removed since this case is covered by the new `interp1d` utility in `partitura.utils.generic`.